test/provisioner/provision_ibmcloud.go

```diff
@@ -344,7 +411,7 @@ func (p *IBMCloudProvisioner) UploadPodvm(imagePath string, ctx context.Context,
 		WithS3ForcePathStyle(true)
 
 	sess := cosession.Must(cosession.NewSession(conf))
-	ibmcloudTrace("session initialized.")
+	log.Info("session initialized.")
```

Comment:
I think this could have the `Debug` level?

Reply:
Looks this could be info level?